Remotion/ObjectBinding/Web/UI/Controls/Validation/IBocListValidationFailureHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// using NUnit.Framework;
+//
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
+
+namespace Remotion.ObjectBinding.Web.UI.Controls.Validation
+{
+  /// <summary>
+  /// The <see cref="IBocListValidationFailureHandler"/> interface provides functionality to handle validation failures.
+  /// </summary>
+  /// <remarks>
+  ///   <note type="inotes">
+  ///     New inheritors of the <see cref="IBocListValidationFailureHandler"/> interface must be registered as seen in <see cref="BocListListValidationFailureHandler"/>.
+  ///     The position should be between -100 and 100.
+  ///   </note>
+  /// </remarks>
+  public interface IBocListValidationFailureHandler
+  {
+    /// <summary>
+    /// Handles validation failures.
+    /// </summary>
+    /// <param name="context">
+    ///   Contains the <see cref="IBocList"/> to handle validation failures for.
+    ///   Should contain the finished error output after failure handling.
+    /// </param>
```

Comment:
```suggestion
```suggestion
    /// <remarks>
    ///   Use <paramref name="context"> to access the <see cref="IBocList"/> and <see cref="IBocListValidationFailureRepository" /> and for reporting the relevant validation failures.
    /// </remarks>
```
```